guides/git/pull_request_process.md

```diff
@@ -0,0 +1,16 @@
+# Process For Working With Pull Requests:
+
+## Overall Process In Several Steps:
+1. If you have just made a commit, GitHub will show an option to make a Pull Request (PR) for the branch you just committed to, near the top of your screen when you are on our repo's main GitHub page. Otherwise, go to the branch you want to make a PR for, click on `Contribute`, then click on `Open Pull Request` to open a PR for your branch.
+2. Add to the description of the PR if need be to properly describe what you did for your PR. This is especially important for frontend tasks, where you can add images and even videos of your progress/functionality.
+3. Assign yourself, as well as others who have worked on the same PR, under `Assignees` (or just remind others who have worked on the PR to add their names).
```

Comment:
Hi Thai, right here! It'll be great if you can add something about how we need to say "closes issue ___" as part of making a PR. just because a few of us (me, yueqi and more) were unaware of it before

Reply:
Oh I don't think that was ever required of any of our team members. Not that it isn't a good practice though. I'll add some words regarding this in the docs